01.multithreading/MultiThreading.Task2.Chaining/Program.cs

```diff
@@ -19,7 +19,7 @@ class Program
         const int TaskAmount = 4;
         public static List<int> taskList = new List<int>();
 
-        static void Main(string[] args)
+        static async Task Main(string[] args)

```

Comment:
let's do this also use `await` functionality instead of synchronous `task.Wait()`